app/client/lib/helpers/helperFunctions.js

```diff
@@ -355,3 +355,17 @@ Helpers.addInputValue = function (inputs, currentInput, formField){
             return value;
         }) || [];
 };
+
+/**
+Takes a camelcase and shows it with spaces
+
+@method toSentence
+@param {string} camelCase    A name in CamelCase or snake_case format
+@return {string} sentence    The same name with spaces
+**/
+Helpers.toSentence = function (name, noHTML) {
```

Comment:
Hm i personally would do it the other way around, as noHTML is "native" HTML is an feature.

E.g. i order to plan for the future, i would make a `Helper.toSentence` ja function, which is can use anywhere and then atop, make it also a template helper. In this case the general helper will never use `toHTML`. Its only interesting for the template helper. Thats why i would make the `noHTML` the default...